lib/cache.js

```diff
@@ -26,22 +25,22 @@ Cache.prototype.add = function(
   this.set(key, val);
 };
 
-Cache.prototype.del = function(
-  // Delete cache element
-  key // string, key
-) {
+// Delete cache element
+//   key - <string>, key
+Cache.prototype.del = function(key) {
   if (this.has(key)) {
     const val = this.get(key);
     this.allocated -= dataSize(val);
   }
   this.delete(key);
 };
 
-Cache.prototype.clr = function(
-  // Clear cache elements that start with prefix
-  prefix, // string, to compare with beginning of the key
-  fn // function (optional), `(key, val)` to be called on each key
-) {
+// Clear cache elements that start with prefix
+//   prefix - <string>, to compare with beginning of the key
+//   fn - <function>, fn(key, val) (optional)
```

Comment:
I propose to change it to `Function` here and everywhere else (similar to how it's done in Node.js docs):
```suggestion
//   fn - <Function>, fn(key, val) (optional)
```